Taskfile

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+if [[ -z "${HISTORIC_TAG_DATA_PATH}" ]]; then
+    echo "[WARNING] You need to define HISTORIC_TAG_DATA_PATH, that will contain historic tag data. Exiting."
+    exit 1
+fi
+
+function build {
+    docker build -t daccs-image-updater:0.1 .
+}
+
+function run {
+    if [[ -z "${GITHUB_USER}" ]]; then
+        echo "[ERROR] Missing GITHUB_USER. Exiting."
+        exit 1
+    fi
+
+    if [[ -z "${GITHUB_PASSWORD}" ]]; then
+        echo "[ERROR] Missing GITHUB_PASSWORD. Exiting."
+        exit 1
+    fi
+
+    docker run -v ${HISTORIC_TAG_DATA_PATH}:/opt/local/src/daccs-image-updater/data \
+     -v ~/.gitconfig:/etc/gitconfig \
+     -e GITHUB_USER=$GITHUB_USER \
+     -e GITHUB_PASSWORD=$GITHUB_PASSWORD \
+     daccs-image-updater:0.1 ./main.sh
+}
+
+function test {
+    docker run -v ${HISTORIC_TAG_DATA_PATH}:/opt/local/src/daccs-image-updater/data \
+     daccs-image-updater:0.1 /bin/sh -c 'cd tests/integration && ./integration_test_runner.sh'
+
+    # -v ~/.gitconfig:/etc/gitconfig \
+}
+
+function clean-data {
+    sudo rm -f ${HISTORIC_TAG_DATA_PATH}/*.old
+    sudo rm -f ${HISTORIC_TAG_DATA_PATH}/*.new
+    sudo rm -f ${HISTORIC_TAG_DATA_PATH}/*.log
+}
+
+function build-test {
+    build
+    test
+}
+
+function build-run {
+    build
+    run
+}
+
+function default {
+    help
+}
+
+function help {
+    echo "Build and test docker image"
+    echo ""
+    echo "Example : HISTORIC_TAG_DATA_PATH=/historic-tag-data $0 build-test"
+    echo ""
+    echo "Actions:"
+    compgen -A function | grep -v help | grep -v default | cat -n
+}
+
+TIMEFORMAT="Task completed in %3lR"
+time ${@:-default}
```

Comment:
Nice trick to use shell script like a poor-man Gradle.  Gradle is heavy (Java) and complex, sometime less is more.